Gopkg.lock

```diff
@@ -13,6 +13,12 @@
   revision = "2f1ce7a837dcb8da3ec595b1dac9d0632f0f99e8"
   version = "v1.3.1"
 
+[[projects]]
```

Comment:
I can't see the usages of newly vendored packages, and many seems very unrelated to this task.

Reply:
yea -- I used a progress bar for a while for the `atxi-build` command, then removed it and haven't pushed the commit removed the deps (oops)